### ISSUE TITLE
(🐞) Wrong error code on bare enum literal

### ISSUE THREAD

**Issue by KotlinIsland** [python/mypy#13236]
```py
from enum import Enum

class E(Enum):
    A = 1
    
a: E.A  # error: Invalid type: try using Literal[E.A] instead?  [misc]
b: 1   # error: Invalid type: try using Literal[1] instead?  [valid-type]
```

**Comment by KotlinIsland** [comment]
Looks like this was resolved.
